tac/project-updates/2025/2025-annual-Hiero.md

```diff
@@ -0,0 +1,82 @@
+These roles are instrumental in guiding the technical direction and ensuring the diverse representation of our community within the Hiero project.
+
+This election will be held on May 7th until May 31st, 2025.
+
+At the moment, we continue to welcome and encorage nominations.
+
+Our current nominees can be viewed [here](https://github.com/hiero-ledger/governance/tree/main/elections/nominees/mar-2025-election)
+
+## Goals
+
+### Performance Against Prior Goals
+
+Our initial target was to move all Hedera repositories to Hiero.
+While we've mostly finished this transition, we've initiated the process of refactoring the code to a more vendor-agnostic and unbranded architecture.
+Our main focus is our SDKs in addition to discussing workflows, roadmaps, and patterns.
+Information and current state on the transition from Hedera can be found [here](https://github.com/hiero-ledger/hiero/blob/main/transition.md).
+Information about our work on making the SDKs vendor neutral can be found [here](https://github.com/hiero-ledger/hiero/blob/main/blog/2025-03-17-sdk-workflows.md).
+
+With all of this work in progress, we plan to release the initial minor versions of all projects in the near future.
+Next, we plan a major release that will contain all the work from moving the code to a more vendor neutral release.
+
+Another main goal is the onboarding of community projects from the Hiero ecosystem.
+At the moment, we moved 3 community projects to Hiero and a new proposal is in the pipeline.
+Information about the onboarding of new projects can be found [here](https://github.com/hiero-ledger/hiero/blob/main/community-transition.md).
+
+On the project's management front, the team is focusing on evaluating and analyzing key project health metrics that track our progress in fostering a thriving and diverse community.
+These metrics enable us to assess the project's performance by measuring contributions, code velocity, and issue response and resolution times.
+
+Our metrics report can be found [here](https://docs.google.com/document/d/e/2PACX-1vRxg-3wU-eg0EWEhAu9SRab_sXwJBy5YDQqoeV6BLq0FnVnoO4V41kPYEDYJovtns6tpq9gkaSaSzlO/pub).
+
+### Next Year's Goals
+
+Hiero is a rapidly growing project that is gaining significant traction. We anticipate continued growth in our community's diversity, enabling the adoption of more community-driven components.
+
+The project is expected to see a considerable growth in the number of maintainers, committers, and members of the TSC.
+
+Our vendor-neutral releases will provide the project with enhanced flexibility and adaptability, supporting its long-term success.
+
+As a project, we expect Hiero to continue advancing in performance, scalability, and, most importantly, trust.
+We remain committed to continue growing our community engagement by encouraging contributions to support both our current and upcoming SDKs, as well as helping to build a more comprehensive TCK with richer test coverage to ensure robust end-to-end validation.
```

Comment:
What does TCK mean?

Reply:
It refers to a Technology Compatibility Kit which is a set of tools used to verify the implementation in our SDKs. I can add better wording to this